### PR TITLE
mds: add to get version in the mds_commands

### PR DESCRIPTION
**Pull request by renhwztetecs**
Sometime we need to determine whether the mds/mon/osd RPM package
is correct, by use the command "ceph tell mds/mon/osd.* version"
is good way.but mds does not support, so I add it.

Signed-off-by: huanwen ren <ren.huanwen@zte.com.cn>

### CODE REVIEW

**Comment by renhwztetecs** [comment]
retest this please

**Comment by renhwztetecs** [comment]
retest this please

**Comment by renhwztetecs** [comment]
ping @batrick  some compiler error

> /home/jenkins-build/build/workspace/ceph-pull-requests-arm64/src/mds/MDSDaemon.cc: In member function ‘int MDSDaemon::_handle_command(const cmdmap_t&, MCommand*, ceph::bufferlist*, std::__cxx11::string*, Context**, bool*)’:
/home/jenkins-build/build/workspace/ceph-pull-requests-arm64/src/mds/MDSDaemon.cc:734:38: error: ‘make_unique’ is not a member of ‘std’
     std::unique_ptr<JSONFormatter> f(std::make_unique<JSONFormatter>());

**Comment by batrick** [comment]
@renhwztetecs, yup. Thanks for the ping.

**Comment by renhwztetecs** [comment]
Also  compiler error, and whether to add the following header file:
#include "common/backport14.h"

**Comment by renhwztetecs** [comment]
retest this please

**Comment by renhwztetecs** [comment]
retest this please

File: src/mds/MDSDaemon.cc [hunk 1]
```diff
@@ -727,27 +727,41 @@ int MDSDaemon::_handle_command(
...
+  std::string format;
+  std::unique_ptr<Formatter> f(Formatter::create(format));
   cmd_getval(cct, cmdmap, "prefix", prefix);
 
   int r = 0;
 
   if (prefix == "get_command_descriptions") {
     int cmdnum = 0;
-    JSONFormatter *f = new JSONFormatter();
+    std::unique_ptr<JSONFormatter> f(ceph::make_unique<JSONFormatter>());
```
Comment (tchaikov) [inline src/mds/MDSDaemon.cc]: nit, could just use `auto` here.

**Comment by renhwztetecs** [comment]
retest this please

**Pull request merged**